File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cs
using System;

public class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        string str = "";
        str += a.ToString();
        str += b.ToString();
        answer = int.Parse(str) > 2 * a * b ? int.Parse(str) : 2 * a * b;
        
        return answer;
    }
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) {
    int answer = 0;
    int num1 = stoi(to_string(a) + to_string(b));
    int num2 = 2 * a * b;
    answer = num1 >= num2 ? num1 : num2;
    return answer;
}